autoload/tagbar.vim

```diff
@@ -2866,6 +2866,46 @@ function! s:EscapeCtagsCmd(ctags_bin, args, ...) abort
     return ctags_cmd
 endfunction
 
+" python simulate system() on window to prevent temporary window creation
+function! s:system_python(cmd, version) abort
+    if has('nvim')
+        let hr = system(a:cmd)
+    elseif has('win32') || has('win64') || has('win95') || has('win16')
```

Comment:
Please re-arrange this if/elseif/else structure to just be if/else. There are two separate cases (the current *if* and the *else* that both have the same outcome, only the *elseif* is different. Just make that the primary test and *else* the other option.

Reply:
This may look like:

```VimL
    if (has('win32') || has('win64') || has('win95') || has('win16')) && has('nvim') == 0
```

, which makes it a little ugly and exceeding 78 characters in one line.

Reply:
Most of these are deprecated flags anyway. Just *win32* should be fine for this purpose. Also there is duplicated code in the version==0 case here, and the has() flags for python versions are never actually _used_.

Reply:
the has('python') is necessary if `a:version` is above zero but user's vim is compiled without python.